simuvex/procedures/syscalls/mmap.py

```diff
@@ -82,10 +82,6 @@ def run(self, addr, length, prot, flags, fd, offset): #pylint:disable=arguments-
             # Can't give you that address. Find a different one.
             addr = self.allocate_memory(size)
```

Comment:
We should probably figure out the address first and then map_region when we know it works so that the memory is always mapped